assessments/0011-keda/keda-analysis.md

```diff
@@ -0,0 +1,416 @@
+
+| Maturity | Requirement | Met? |
+| ---      | ---         | --- |
+| Sandbox  | Majority of [Website guidelines][website-guidelines] satisfied | Yes |
+| Sandbox | [Docs assessment][assess-howto] [submitting a request][cncf-servicedesk] completed. | Yes |
+| Sandbox | **Project documentation** exists – somewhere. It is acceptable at this maturity level to link out to documentation that hasn't yet been integrated into the website. (May still be in the project GitHub repo, for example.) | Yes |
+| Incubating | All [Website guidelines][website-guidelines] satisfied | Yes   |
+| Incubating | Request docs (re-)assessment through CNCF [service desk][cncf-servicedesk] | Yes |
+| Incubating | **Project doc**: stakeholders (roles) identified and doc needs documented | No |
+| Incubating | **Project doc**: Hosted directly | Yes |
+| Incubating | **Project doc**: Comprehensive, addressing most stakeholder needs | Yes |
+| Graduated | [Docs assessment][assess-howto]: all assessment follow-through actions are complete | No |
+| Graduated | **Project doc** fully addresses needs of key stakeholders | No - new user doc needs improvement |
+| Archived | The website repo is in an [archived state][archiving-repo] | n/a |
+| Archived | Archived status of the project is obvious to site visitors | n/a |
+| Archived | Link to successor project website and/or migration documentation (if it exists) | n/a |
+
+### Usability, accessibility and devices
+
+The website is **usable from mobile**. Top-nav is reachable only via the hamburger menu on mobile devices. There is no in-page TOC or other context on mobile devices. The footer is identical on all platforms.
+
+**Search** is available from the hamburger menu as well.
+
+**Doc pages are readable** on all platforms.
+
+A **mobile-first design** does not make sense for this project.
+
+**Color contrasts** are mostly adequate. Some blue-on-black graphics and text are probably hard to read for color- or contrast-impaired readers.
+
+Website features, *Search* most importantly, are usable using a **keyboard only**.
+
+As with any text that contain a lot of code and special characters, **text-to-speech** is **not likely to offer listeners a good experience**.
+
+### Branding and design
+
+The website and documentation carry an **easily recognizable brand** for the project based on logo, color scheme, and template layout. The **brand is used consistently** on the site. The website's text is **easily readable**.
+
+
+### Case studies/social proof
+
+
+I'm unable to find **case studies** or **user testimonials** for the project. They're probably not as important for KEDA as they are for a more extensive product, though.
+
+There is a **project blog**; posts are infrequent. The last one was in August 2023.
+
+There are **community talks** for the project on YouTube. One talk, from KubeCon 2022, is **present on the website**.
```

Comment:
This is a good callout - We should link to your talk from KubeCon NA @zroubalik @JorTurFer until 2024 EU is live